execution_engine/src/storage/trie_store/operations/mod.rs

```diff
@@ -706,16 +700,13 @@ where
 fn rehash<K, V>(
     mut tip: Trie<K, V>,
     parents: Parents<K, V>,
-) -> Result<Vec<(Digest, Trie<K, V>)>, bytesrepr::Error>
+) -> Result<Vec<(Digest, Trie<K, V>)>, TrieHashingError>
 where
     K: ToBytes + Clone,
     V: ToBytes + Clone,
 {
     let mut ret: Vec<(Digest, Trie<K, V>)> = Vec::new();
-    let mut tip_hash = {
-        let node_bytes = tip.to_bytes()?;
-        Digest::hash(&node_bytes)
-    };
+    let mut tip_hash = tip.hash()?;
```

Comment:
This kind of messes up my IDE... there's the standard library trait method [`std::hash::Hash::hash`][1] which collides with this hash method.

[1]: https://doc.rust-lang.org/std/hash/trait.Hash.html#tymethod.hash

Reply:
Whoops. Should I rename it? (And if yes, any suggestions what to rename it to?)

Reply:
`digest_hash` maybe?

Reply:
Or `trie_hash`?